blynklib_mp.py

```diff
@@ -203,6 +218,13 @@ def _get_socket(self):
             self._socket = socket.socket()
             self._socket.connect(socket.getaddrinfo(self.server, self.port)[0][-1])
             self._set_socket_timeout(self.SOCK_TIMEOUT)
+            if self.port == 443 or self.port == 8443:
+                self.log('Using SSL socket...')
+                self._socket = ssl.wrap_socket(self._socket)
+                # Short reads are not supported in ssl mode. We work around
```

Comment:
Better to remove comments especially for MP
cause out target small lib size ))

Reply:
Understood, will do.